src/azul/service/manifest_service.py

```diff
@@ -1007,8 +1007,29 @@ def source_filter(self) -> SourceFilters:
     @classmethod
     def command_lines(cls, url: str) -> JSON:
         return {
-            'cmd.exe': f'curl.exe --location {cls._cmd_exe_quote(url)} | curl.exe --config -',
-            'bash': f'curl --location {shlex.quote(url)} | curl --config -'
+            # Normally, curl writes the response body and returns 0 (success),
+            # even on server errors. With `--fail`, it writes an error message
```

Comment:
```suggestion
            # even on server errors. With --fail, it writes an error message
```

Since that's what you chose in the other PR.